src/renderer/src/components/TransactionActivityList/index.tsx

```diff
@@ -228,7 +233,9 @@ const Content = ({ defaultAccounts }: TProps) => {
                       transform: `translateY(${virtualItem.start}px)`,
                     }}
                   >
-                    <h3 className="flex h-10 max-h-10 min-h-10 items-center font-medium text-white">{date}</h3>
+                    <h3 className="flex h-10 max-h-10 min-h-10 items-center font-medium text-white">
+                      {DateHelper.formatLocalized(date, t('dateRange.formatExtendedDate'), locale)}
```

Comment:
What do you think to create a hook for these date format? So you won't need to use the `useTranslation` in each place.